src/components/Accordion/Accordion.css.ts

```diff
@@ -35,6 +35,7 @@ export const BodyUI = styled('div')`
   ${baseStyles};

```

Comment:
In https://github.com/helpscout/hsds-react/pull/613, this file has been moved to its own `styles` folder... might want to do it here to avoid merge conflicts 👍

Reply:
@tinkertrain Will do x 2